style.css

```diff
@@ -0,0 +1,102 @@
+*,
+*::before,
+*::after {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  list-style: none;
+}
+
+body {
+  width: 375px;
+}
+
+/* header part css */
+
+header {
+  background: #3c3a39;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.navbar img {
+  margin: 24px 0 12px 20px;
+  cursor: pointer;
+}
+
+.navbar .toggler {
+  margin: 24px 38px 19px 0;
+}
+
+/* main part css */
+
+main {
+  background: #1c1a19;
+}
+
+article {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.main h2 {
+  padding-top: 120px;
+  width: 80%;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+}
+
+.main h3 {
+  padding-top: 12px;
+  width: 80%;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  color: #fff5e1;
+}
+
+.main p {
+  padding-top: 12px;
+  width: 80%;
+  font-family: 'Poppins', sans-serif;
```

Comment:
You are using a font-family that you haven't imported. To import font `Poppins` go [there](https://fonts.google.com/specimen/Poppins). 
| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/219129280-de989ae5-2459-4051-901b-32299823e384.png)  | ![image](https://user-images.githubusercontent.com/61361037/219129384-21377059-c80d-482d-acc0-a0888e75308d.png)  |